Kickstarter-iOS/Views/SettingsGroupedFooterView.swift

```diff
@@ -0,0 +1,45 @@
+import Library
+import Prelude
+import UIKit
+
+final class SettingsGroupedFooterView: UITableViewHeaderFooterView {
+  // MARK: - Properties
+  
+  private(set) lazy var label: UILabel = {
+    UILabel(frame: .zero)
+      |> settingsHeaderFooterLabelBaseStyle
```

Comment:
The base style has to be available in `init` so that the auto layout & dynamic sizing is established correctly. The first time we're instantiating this lazy property is in `init` when we call `self.contentView.addSubviewConstrainedToMargins(self.label)`  I'm not entirely sure why at this point but spend enough time on this so we should accept this and maybe look into it at some time in the future.